src/Microsoft.DotNet.Configurer/BashPathUnderHomeDirectory.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+
+namespace Microsoft.DotNet.Configurer
+{
+    public struct BashPathUnderHomeDirectory
```

Comment:
Should we name this somewhat more generic? What if, in the future we decide to not put our artifacts (or all of them) under the home folder? I guess it is ok for now. We can rename later if needed.

Reply:
Like the comment below. I just cannot find a good abstraction in order to make it generic. Bash as $HOME and ~ while cmd has only %HOME%